src/index.js

```diff
@@ -243,7 +239,7 @@ async function main(args) {
       { level: 'debug', args },
     )
     const result = await webHelpers.uploadToCodecovPUT(uploadURL, gzippedFile)
-    log(result)
+    log(JSON.stringify(result))
```

Comment:
I'm really starting to wish I went with a logger like winston then rolling my own logger. Needs the balance between sanity and file size. 🤔